index.html

```diff
@@ -1,59 +1,284 @@
+          </div>
+        </article>
+
+        <article class="project-articles">
+          <div class="textside">
+            <h2>Multi-Post Stories</h2>
+
+            <ul class="card-ul">
+              <li>FACEBOOK</li>
+              <li><img src="./images/counter.png" /></li>
+              <li>Full Stack Dev</li>
+              <li><img src="./images/counter.png" /></li>
+              <li>2015</li>
+            </ul>
+            <p>
+              Experiment content creation feature that allows users to add an
+              existing story over the course of a day without spamming their
+              friends
+            </p>
+            <ul class="skills">
+              <li>html</li>
+              <li>Ruby on Rails</li>
+              <li>css</li>
+              <li>javascript</li>
+            </ul>
+            <button class="see-project">See Project</button>
+          </div>
+          <div class="card">
+            <img src="./images/snapshot2.png" />
+          </div>
+        </article>
+
+        <article class="project-articles">
+          <div class="card">
+            <img src="./images/snapshot3.png" />
+          </div>
+          <div class="textside">
+            <h2>Facebook 360</h2>
+            <ul class="card-ul">
+              <li>FACEBOOK</li>
+              <li><img src="./images/counter.png" /></li>
+              <li>Back End Dev</li>
+              <li><img src="./images/counter.png" /></li>
+              <li>2015</li>
+            </ul>
+            <p>
+              Experiment content creation feature that allows users to add an
+              existing story over the course of a day without spamming their
+              friends
+            </p>
+            <ul class="skills">
+              <li>html</li>
+              <li>Ruby on Rails</li>
+              <li>css</li>
+              <li>javascript</li>
+            </ul>
+            <button class="see-project">See Project</button>
+          </div>
+        </article>
+
+        <article class="project-articles">
+          <div class="textside">
+            <h2>Tonic</h2>
+            <ul class="card-ul">
+              <li>CANOPY</li>
+              <li><img src="./images/counter.png" /></li>
+              <li>Back End Dev</li>
+              <li><img src="./images/counter.png" /></li>
+              <li>2015</li>
+            </ul>
+            <p>
+              Experiment content creation feature that allows users to add an
+              existing story over the course of a day without spamming their
+              friends
+            </p>
+            <ul class="skills">
+              <li>html</li>
+              <li>Ruby on</li>
+              <li>css</li>
+              <li>javascript</li>
+            </ul>
+            <button class="see-project">See Project</button>
+          </div>
+          <div class="card">
+            <img src="./images/snapshot4.png" />
+          </div>
+        </article>
+      </section>
```

Comment:
Nice work building each card for the project section😉😉 But we can make it closer to the given [design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=YsoN9tL1LV5poZo5-0) don't forget it is a requirement that we stick to the design in this milestone😊😊

your project😉
<img width="337" alt="your cards" src="https://user-images.githubusercontent.com/64643995/220052032-5930b34b-d9c1-4ff7-9be8-524468e7ef9a.png">


design😊
<img width="230" alt="design" src="https://user-images.githubusercontent.com/64643995/220052330-e5811a27-3f28-430d-a5e7-134ece65c38d.png">

kindly take note of the places highlighted😉 ensure to take note of the spacings, font-properties and colors as you make your changes and ensure to stick to the design😊😊